### ISSUE TITLE
Help offer: Add support for Chroma 1.6/2.1/3.7/6.0" tags

### ISSUE THREAD

**Issue by zendes** [OpenEPaperLink/OpenEPaperLink#356]
I've recently acquired a batch of Chroma SubGhz tags, and noticed that not all of them have firmware available. I have successfully flashed Chroma 2.9, 4.2 and 7.4" tags, connected to a homebuilt Yellow AP with an EByte CC1101 module, but also have one of each of the following sizes: 1.6", 2.1", 3.7" and 6.0". I also got a different one called Aura 42, which seems to use the same pcb but a different epaper display. I'll attach photos of all the internals.

If people (probably @skiphansen) are interested in working together on adding support for these tags to the codebase, I'm happy to help. I have debug equipment such as a logic analyzer and can pull/build code, but I haven't worked with this microcontroller or epaper displays before. Let me know if this is something you are interested in and have time for.

![IMG_6976](https://github.com/user-attachments/assets/e9ee531b-89d3-407e-a41e-b3bde86ce9d3)
![IMG_6975](https://github.com/user-attachments/assets/97a707bf-54ac-4d2f-a7f7-43078423ae9b)

![IMG_6970](https://github.com/user-attachments/assets/e24b9a5f-693c-4a33-a903-80fcf9ce9778)
![IMG_6969](https://github.com/user-attachments/assets/0da5bb82-b865-4c11-ac7d-b51d58803c71)

![IMG_6972](https://github.com/user-attachments/assets/6949f714-733b-4751-b741-42dfbfe56f13)
![IMG_6971](https://github.com/user-attachments/assets/d35ad5fa-669b-4739-b481-e3ff389f06d7)

![IMG_6973](https://github.com/user-attachments/assets/47cd2a68-0721-4c35-aa39-4f1ecf0ba2f8)
![IMG_6974](https://github.com/user-attachments/assets/71bb6948-a759-4467-8d66-26b4a43dac5a)

![IMG_6977](https://github.com/user-attachments/assets/d493bae7-4a5d-44b4-9dc7-0e93649ea751)
![IMG_6978](https://github.com/user-attachments/assets/326d5c70-e435-4c31-bbd8-00246cabf33f)

**Comment by skiphansen** [comment]
@zendes Yes, I am very interested in working with you!  I have a number of Chroma 21s and that is my next target, but you have several other tags that I don't have examples of.  Let's chat on discord.

**Comment by skiphansen** [comment]
To get you started here's some [documentation](https://github.com/skiphansen/dmitrygr-einkTags/blob/master/docs/Chroma29.md#logic-analyzer-connections-for-epd-reverse-engineering) on how I connected my logic analyzer to the Chroma29 when I reverse engineered it.

I recently bought a "spider board" to make it easier to reverse engineer tags without a bunch of soldering. I'm waiting for it to arrive before tackling the Chroma21.

The first thing to do is backup the stock firmware.  This is easy since the device is not read protected.  To void possible copyright issues please do not post them publicly.  Once backed up please add the MD5 signature of the stock fw to the Wiki [here](https://github.com/OpenEPaperLink/Tag_FW_Chroma/wiki/Known-stock-firmware-images).

The next thing to do is to backup the factory EEPROM image.  If you run Linux it's easy "ish" using my chroma_shell tool and proxy firmware. Documentation is [here](https://github.com/skiphansen/dmitrygr-einkTags/tree/master/chroma_shell#readme).

**Comment by zendes** [comment]
thanks. I'll try to dump the firmware on all of them this weekend, and report back if I have any issues. so far I've just opened them to see the logic board, and all of them are still running stock firmware

**Comment by pandaboy6621** [comment]
Any updates for the Chroma 21? I have two of these tags.

**Comment by skiphansen** [comment]
@pandaboy6621 I got distracted by other projects.  You're only the second person that I know of that has any, if more people
had them it would be more worthwhile.

**Comment by uszke** [comment]
I have at least 250 of Chroma 21. Probably closer to something like 400-500, never went through with counting them to be honest.
Any way I could contribute to getting them up and running? I could also send some to the core devs.

**Comment by skiphansen** [comment]
Cool! Will you be making any of your excess available to others in the group?  If so that makes the effort more worthwhile.

A good first step would be to update the [wiki](https://github.com/OpenEPaperLink/OpenEPaperLink/wiki/2.1%E2%80%B3-Chroma%E2%80%9021#history) with what you have if the data isn't already present.

The first 4 characters of the SN will tell us a lot.

**Comment by uszke** [comment]
I was planning on selling them, but my morals don't allow me to do that if I'm not certain that every piece is working and I don't have the original programmer to check that easily.
That being said, I'm pretty sure that I can share some for free with someone that's interested in porting them.
We can arrange something on discord I guess?

**Comment by skiphansen** [comment]
There's nothing wrong with selling them for "parts only" at the right price as long as you don't represent them as working.

Programmers are cheap, but unless they are the newest versions the programming pads are not accessible w/o taking the tag apart.  At least you don't have to guess the location and drill holes!

What are the first 4 characters of the SN?  Are all the same?  If you have the same tags that I have then I don't need any, but if they are different I'd be interested in a few examples.

**Comment by uszke** [comment]
Alright, I'll keep that in mind, thank you!

As for the serial numbers, I have JB00, JB10 and JB11. Few pieces of the last one, from what I can see
